not-to-release/sources/newsgroup/groups.google.com_CaptSpasticsJokes_4c06327c7fb39291_ENG_20050727_125600.xml.conllu

```diff
@@ -49,7 +49,7 @@
 2-3	Spastic's	_	_	_	_	_	_	_	_
 2	Spastic	Spastic	PROPN	NNP	Number=Sing	5	nmod:poss	5:nmod:poss	_
 3	's	's	PART	POS	_	2	case	2:case	_
-4	Demented	Demented	PROPN	NNP	Number=Sing	5	compound	5:compound	_
+4	Demented	Dement	VERB	NNP	Tense=Past|VerbForm=Part	5	amod	5:amod	_
```

Comment:
Not sure this is really VERB

Reply:
Happy to change it to ADJ (though I don't think EWT is particularly consistent about the VERB/ADJ boundary)